src/main.rs

```diff
@@ -424,6 +425,44 @@ fn set_trace_id() {
   std::env::set_var(key, my_id)
 }
 
+fn cmd_info(
+    _config: Arc<Config>,
+    target_path: PathBuf,
+    _diff: bool,
+    _overview: bool,
+    _branch: bool,
+    _local: bool
```

Comment:
Check that diff/overview/branch are false, and local is true, and error out if not, since we haven't implemented them yet? (Or maybe just remove diff/overview/branch for now, and let it just error out as unknown arguments)